lib/template.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * replaces all uppercase dollar templates with their string representation from obj
+ * if replacement is undefined in obj the dollar template string is left untouched
+ */
+
+const template = (string, obj) => {
```

Comment:
I could not get lodash template to behave how I wanted, so I wrote this small function